README.md

```diff
@@ -36,6 +36,7 @@ A more complex example
 * `strict` (Default: `False`) will raise an exception if unsupported features are found in the obj or mtl file
 * `encoding` (Default: `utf-8`) of the obj and mtl file(s)
 * `create_materials` (Default: `False`) will create materials if mtl file is missing or obj file references non-existing materials
+* `consume_faces` (Default: `False`) will consume a face list for every material as well. Beware that it will greatly increase memory usage for large models.
```

Comment:
I'm not sure if "consume" is an understandable word to use here. If I didn't know the details of the parser I would be confused about this option.

Reply:
Personally I liked "collect" much better. "consuming" is just a parser term of processing data and not necessarily related to what we do with the data itself.

Reply:
Sure, renamed it back.